AmericanMedicalAssociation/AMA-style-guide
# Ama 76 homepage template

https://palantir.atlassian.net/browse/AMA-76

Steps to test: 
- navigate to Templates -> Home -> Home
- observe that the Home Template pattern exists
- observe that the Promo Block organism is responsive and has proper type sizing
- observe that the Page-Level Search molecule exists and has proper responsive type sizing
- observe that the overall Patternlab scaffolding and headings have margin padding for Atoms, Molecules, and Organisms, but not for Templates
